roachpb/data.go

```diff
@@ -784,6 +784,11 @@ func (s Span) Equal(o Span) bool {
 	return bytes.Equal(s.Key, o.Key) && bytes.Equal(s.EndKey, o.EndKey)
 }
 
+// ContainsKey returns whether this span contains the specified key.
+func (s Span) ContainsKey(key Key) bool {
```

Comment:
I would advise against having comparison functions on `Key` types because of local addressing. If you're trying to use `ContainsKey`, chances are that you want to be looking at an `RSpan`.